### PR TITLE
Feature/fetch hosted mock data

### PR DESCRIPTION
**Pull request by LTDakin**
New
- Function loadUserImages that fetches all frames with an authtoken from the archive, called on component mount and stores frames in a ref named UserFrames
- ref userFrames that holds the user's images from the archive
- A skeleton loader that is displayed when we don't have the image data on hand yet

Changed
- updating property names so that images will be working from the new format instead of the mock data format
- pointing the datalab api url to the server link instead of a locally running instance
- removed the filter and map step when displaying data session images so that the key and alt text can be the file name instead of an index for future image operation selection.

Fixes
- missing image alt text
- binding of transition to a number with v-bind instead of treating the 0 as a string which caused warnings

Side note: we have updated the datalab url to point to the server api instead of a local one. However the original implementation didn't use the fetchApi util and so Data Session View is non functional. A fix updating all of those api requests to use the store url and api util function will fix this.

Screenshot 2024-01-29 at 2 15 43 PM

### CODE REVIEW

File: src/views/ProjectView.vue [hunk 1]
```diff
@@ -27,7 +24,13 @@ const authHeaders = {
 
 // toggle for optional data viewing, controlled by a v-switch
 let imageDisplayToggle = ref(true)
+let userFrames = ref(null)
 
+// Loads the user's Images from their profile into userImages ( currently just fetches all frames from archive regardless of proposal )
+const loadUserImages = async () => {
+    const url = archiveUrl + 'frames/?reduction_level=95'
+    userFrames.value = await fetchApiCall({url: url, method: 'GET', token: 'Token 2da74cb5262a52bc479e5b63b548fd5910592475'})
```
Comment (capetillo) [inline src/views/ProjectView.vue:32]: use `store.state.authToken`

**Review by capetillo — COMMENTED** [review]
hopefully you can read my comment

File: src/views/ProjectView.vue (hunk 1 above)
Comment (LTDakin) [reply src/views/ProjectView.vue]: oops! good catch

File: src/views/ProjectView.vue [hunk 2]
```diff
@@ -27,7 +24,13 @@ const authHeaders = {
 
 // toggle for optional data viewing, controlled by a v-switch
 let imageDisplayToggle = ref(true)
+let userFrames = ref(null)
 
+// Loads the user's Images from their profile into userImages ( currently just fetches all frames from archive regardless of proposal )
+const loadUserImages = async () => {
```
Comment (mgdaily) [inline src/views/ProjectView.vue:30]: Suggestion: We could put the reduction level as an argument to this function so that it's more reusable

**Review by mgdaily — APPROVED** [review]
One suggestion/comment. Otherwise looks good!

**Pull request merged**